nifi/pom.xml

```diff
@@ -789,6 +789,11 @@
                 <artifactId>nifi-geo-nar</artifactId>
                 <version>0.1.0-incubating-SNAPSHOT</version>
                 <type>nar</type>
+            <dependency>
```

Comment:
this change misses closing the geo-nar dependency line, which breaks compilation.